CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented in this file.
 
+### [TO BE DEFINED] 2019.04.26
```

Comment:
could you kindly set the version number to: `1.2.8` and leave the date with `TBD`?
thanks